api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -326,9 +326,16 @@ private int getBufferMaxCapacity(String accessTierShortName) {
   }
 
   public BillingProjectBufferStatus getStatus() {
-    final long bufferSize =
-        billingProjectBufferEntryDao.countByStatus(
-            DbStorageEnums.billingProjectBufferEntryStatusToStorage(BufferEntryStatus.AVAILABLE));
-    return new BillingProjectBufferStatus().bufferSize(bufferSize);
+    return new BillingProjectBufferStatus()
+        .availablePerTier(
+            accessTierService.getAllTiers().stream()
+                .map(
+                    tier ->
+                        new AvailableBufferPerTier()
+                            .accessTierShortName(tier.getShortName())
+                            .bufferSize(
+                                billingProjectBufferEntryDao.countByStatusAndAccessTier(
```

Comment:
Can we make this in one query, i.e. group by status and tier?

Reply:
good idea.  done.